trtools/testsupport/sample_vcfs/dumpSTR_vcfs/locus_filters.vcf

```diff
@@ -439,7 +439,7 @@ chr21	14383027	GANGSTR_437028	TTTTGTTGTTGTTGTTG	TGTTGTTGTTGTTGTTG	.	HETLOW0.05;f
 chr21	14390199	GANGSTR_437030	TGGTGAGTGTGTGTGTGTGT	TAGTGAGTGTGTGTGTGTGT	.	HETLOW0.05;foo_region	INFRAME_PGEOM=0.95;INFRAME_UP=0.01;INFRAME_DOWN=0.01;OUTFRAME_PGEOM=0.95;OUTFRAME_UP=0.01;OUTFRAME_DOWN=0.01;START=14390205;END=14390218;PERIOD=2;NSKIP=0;NFILT=0;BPDIFFS=0;DP=222;DSNP=0;DSTUTTER=2;DFLANKINDEL=7;AN=6;REFAC=3;AC=3;HRUN=2;HET=0;HWEP=1	GT:GB:Q:PQ:DP:DSNP:DSTUTTER:DFLANKINDEL:PDP:PSNP:GLDIFF:AB:DAB:FS:ALLREADS:MALLREADS:FILTER	0|1:0|0:1:0.5:77:0:1:2:48.14|28.86:0|0:66.66:-1.65:63:-0.96:-2|1;0|57:0|53:PASS	0|1:0|0:1:0.5:67:0:1:4:54.58|12.42:0|0:5.56:-8.49:54:-0.65:-2|1;0|50:0|42:PASS	0|1:0|0:1:0.5:78:0:0:1:54.02|23.98:0|0:44.75:-3.64:64:-0.4:0|55:0|53:PASS
 chr21	14394189	GANGSTR_437031	ACAAAAACAAACAAACAAAC	AAAACAAACAAACAAAC	.	CALLRATE0.5;foo_region;NO_CALLS_REMAINING	INFRAME_PGEOM=0.59;INFRAME_UP=0.02;INFRAME_DOWN=0;OUTFRAME_PGEOM=0.69;OUTFRAME_UP=0;OUTFRAME_DOWN=0.01;START=14394193;END=14394208;PERIOD=4;NSKIP=0;NFILT=3;BPDIFFS=-3;DP=0;DSNP=0;DSTUTTER=0;DFLANKINDEL=0;AN=0;REFAC=0;AC=0;HRUN=5;HET=-1;HWEP=-1	GT:GB:Q:PQ:DP:DSNP:DSTUTTER:DFLANKINDEL:PDP:PSNP:GLDIFF:AB:DAB:FS:ALLREADS:MALLREADS:FILTER	.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:NOCALL	.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:NOCALL	.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:NOCALL
-chr21	14405799	GANGSTR_437032	ACCCACCCACCC	ACCCGCCCACCC	.	HWE0.5;HETHIGH0.45	INFRAME_PGEOM=0.95;INFRAME_UP=0;INFRAME_DOWN=0;OUTFRAME_PGEOM=0.95;OUTFRAME_UP=0;OUTFRAME_DOWN=0;START=14405799;END=14405810;PERIOD=4;NSKIP=0;NFILT=0;BPDIFFS=0;DP=342;DSNP=0;DSTUTTER=0;DFLANKINDEL=1;AN=6;REFAC=3;AC=3;HRUN=3;HET=0.5;HWEP=0.25	GT:GB:Q:PQ:DP:DSNP:DSTUTTER:DFLANKINDEL:PDP:PSNP:GLDIFF:AB:DAB:FS:ALLREADS:MALLREADS:FILTER	0|1:0|0:1:0.5:115:0:0:1:59.46|55.54:0|0:179.46:-0.11:104:-0.16:0|87:0|86:PASS	0|1:0|0:1:0.5:109:0:0:0:53.35|55.65:0|0:152.82:-0.04:98:-0.53:0|83:0|81:PASS	0|1:0|0:1:0.5:118:0:0:0:74.97|43.03:0|0:123.77:-2.66:104:0:0|82:0|78:PASS
+chr21	14405799	GANGSTR_437032	ACCCACCCACCC	ACCCGCCCACCC	.	HWE0.5;HETHIGH0.45;foo_region	INFRAME_PGEOM=0.95;INFRAME_UP=0;INFRAME_DOWN=0;OUTFRAME_PGEOM=0.95;OUTFRAME_UP=0;OUTFRAME_DOWN=0;START=14405799;END=14405810;PERIOD=4;NSKIP=0;NFILT=0;BPDIFFS=0;DP=342;DSNP=0;DSTUTTER=0;DFLANKINDEL=1;AN=6;REFAC=3;AC=3;HRUN=3;HET=0.5;HWEP=0.25	GT:GB:Q:PQ:DP:DSNP:DSTUTTER:DFLANKINDEL:PDP:PSNP:GLDIFF:AB:DAB:FS:ALLREADS:MALLREADS:FILTER	0|1:0|0:1:0.5:115:0:0:1:59.46|55.54:0|0:179.46:-0.11:104:-0.16:0|87:0|86:PASS	0|1:0|0:1:0.5:109:0:0:0:53.35|55.65:0|0:152.82:-0.04:98:-0.53:0|83:0|81:PASS	0|1:0|0:1:0.5:118:0:0:0:74.97|43.03:0|0:123.77:-2.66:104:0:0|82:0|78:PASS
```

Comment:
The PR seems good overall. Can you just explain briefly why the output is changing? Was it wrong in the first place?

Reply:
ah, oops - see [my (now updated) original comment](https://github.com/gymrek-lab/TRTools/pull/184#issuecomment-1803002481) about it. I think pysam might treat end coordinates as exclusive of the region while pybedtools treats them inclusive. But it could also be due to other things like 0-based vs 1-based indexing

I'm going to convert this PR back into a draft until I have some time to look through this and figure out what's going on

Reply:
Okay, I'm okay with that. Officially, bed files are end-point exclusive, so I don't mind treating them that way, that seems right, we should just specify that in our docs.